configuration/components/observatorium.libsonnet

```diff
@@ -102,6 +102,58 @@ local api = (import 'observatorium-api/observatorium-api.libsonnet');
       querier: 1,
       query_frontend: 1,
     },
+    resources: {
+      compactor: {
+        requests: {
+          memory: '100Mi',
+          cpu: '100m',
+        },
+        limits: {
+          memory: '200Mi',
+          cpu: '200m',
+        },
+      },
+      distributor: {
+        requests: {
+          memory: '100Mi',
+          cpu: '100m',
+        },
+        limits: {
+          memory: '200Mi',
+          cpu: '200m',
+        },
+      },
+      ingester: {
+        requests: {
+          memory: '100Mi',
+          cpu: '100m',
+        },
+        limits: {
+          memory: '200Mi',
+          cpu: '200m',
+        },
+      },
+      querier: {
+        requests: {
+          memory: '100Mi',
+          cpu: '100m',
+        },
+        limits: {
+          memory: '200Mi',
+          cpu: '200m',
+        },
+      },
+      query_frontend: {
+        requests: {
+          memory: '100Mi',
+          cpu: '100m',
+        },
+        limits: {
+          memory: '200Mi',
+          cpu: '200m',
+        },
+      },
+    },
```

Comment:
The above change looks like duplicating defaults which are already included in [loki.libsonnet](https://github.com/observatorium/observatorium/blob/main/configuration/components/loki.libsonnet#L32). AFAICS there is a small bug introduced by @kakkoyun's refactoring [here](https://github.com/observatorium/observatorium/blob/main/configuration/components/loki.libsonnet#L20). 

My suggestion is to remove [loki.libsonnet#L20-21]() and fix the [loki.libsonnet#L321](https://github.com/observatorium/observatorium/blob/main/configuration/components/loki.libsonnet#L321) and [loki.libsonnet#L388](https://github.com/observatorium/observatorium/blob/main/configuration/components/loki.libsonnet#L388) to use the `components...resources` fields.